cmd/root.go

```diff
@@ -34,12 +42,25 @@ HPC deployments on the Google Cloud Platform.`,
 				log.Fatalf("cmd.Help function failed: %s", err)
 			}
 		},
-		Version: "v1.4.1",
+		Version:     "v1.4.1",
+		Annotations: annotation,
 	}
 )
 
 // Execute the root command
 func Execute() error {
+	if len(GitBranch) > 0 {
```

Comment:
I would use `GitCommitInfo` here instead. It is possible that they are using git but not on any branch (detached HEAD mode). This is possible if you do `git checkout <sha>`.